MonitorizareVot/Onboarding/Controllers/AddNotesViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AddNotesViewController.swift
+//  MonitorizareVot
+//
+//  Created by Geart Otten on 13/10/2019.
+//  Copyright © 2019 Code4Ro. All rights reserved.
+//
+
+import UIKit
+
+class AddNotesViewController: UIViewController {
+
+    // MARK: Variables
+    
+    private var onboardingView: OnboardingView!
+    
+    private var onboardingViewModel = OnboardingViewModel(title: "Add notes",
+                                                          context: "Did you notice any problem that does not appear in the form? Use the ‘Add note’ button, write a message and/or attach a picture.",
+                                                          imageString: "3")
+
+    // MARK: Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
```

Comment:
Please see the `UIColor` extension that has the colors we should use. If there's a color that's not there, please add it there first and use it in all other places. This way, when we decide to switch the colors, we won't have to go through all the project